### PR TITLE
docs: update repos and pipelines docs

### PR DESCRIPTION
**Pull request by alaincormier-ssc**
update repositories and pipelines documents for clarity and new repo templates

### CODE REVIEW

**Review by seanwilkens-ssc — DISMISSED** [review]

File: Landing Zone Operations/Repositories.md [hunk 1]
```diff
@@ -166,23 +179,23 @@ These other policies can also be enabled as needed:
 
 An AzDO service account should be used for authenticating Config Sync.  It requires read access to the monorepo.
 
-Depending on your organization, this could be set at different levels and with groups.  These steps will assume the service account has already been configured.
+Depending on your organization, this could be set at different levels and with groups.  These steps assume the service account has already been configured.
```
Comment (Michael-McAuley-SSC) [inline Landing Zone Operations/Repositories.md:182]: for clarity - could you elaborate that this as an azdo service account?

**Review by Michael-McAuley-SSC — COMMENTED** [review]
Looks good, just a couple small comments/suggestions

File: Landing Zone Operations/Pipelines.md [hunk 2]
```diff
@@ -34,29 +36,31 @@ However, its name defaults to the repo's name which is not ideal if the repo con
 Click the **"three dots" > Rename/move** to provide a meaningful name and optionally move to a folder.
 For example, if your repo's name is `my-repo` and the pipeline YAML file is `firstpipeline.yaml`, the pipeline could be renamed to something like `my-repo__firstpipeline` in a `my-repo` folder.
 
-> *During a pipeline’s first execution, it may be required to manually authorize it to use agent pools (depending on security settings) and/or access other resources (repos, variable groups, etc.).*
+> *Note: During a pipeline’s first execution, it may be required to manually authorize it to use agent pools (depending on security settings) and/or access other resources (repos, variable groups, etc.).*
```
[the remark this answers is not in the document]
Comment (alaincormier-ssc) [reply Landing Zone Operations/Pipelines.md]: updated
please review / resolve comment

File: Landing Zone Operations/Repositories.md (hunk 1 above)
Comment (alaincormier-ssc) [reply Landing Zone Operations/Repositories.md]: updated
please review / resolve comment

**Review by Michael-McAuley-SSC — APPROVED** [review]

**Pull request merged**